pyproject.toml

```diff
@@ -274,6 +276,11 @@ archs = ["x86_64", "arm64"]
 [tool.cibuildwheel.linux]
 archs = ["auto", "aarch64"]
 
+[[tool.cibuildwheel.overrides]]
+# Install nightly wheels for dependencies not yet available on PyPI.
+select = "cp314*"
+before-test = "python -m pip install --pre --extra-index-url https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy --only-binary numpy"
```

Comment:
this might become obsolete sooner than I expected: https://github.com/numpy/numpy/pull/29438
happy to revisit tomorrow and see where things are then

Reply:
wow, it is actually being published right now: https://pypi.org/project/numpy/#files
I'll update the PR immediately then